source/Octopus.Versioning/Docker/DockerTag.cs

```diff
@@ -43,33 +43,8 @@ public DockerTag(OctopusVersion version)
         {
         }
 
-        public override int Major => IsLatest ? int.MaxValue : base.Major;
-        public override int Minor => IsLatest ? int.MaxValue : base.Minor;
-        public override int Patch => IsLatest ? int.MaxValue : base.Patch;
-        public override int Revision => IsLatest ? int.MaxValue : base.Revision;
-
-        bool IsLatest => base.Major == 0 &&
-            base.Minor == 0 &&
-            base.Patch == 0 &&
-            base.Revision == 0 &&
-            base.ReleasePrefix == Latest &&
-            string.IsNullOrEmpty(base.ReleaseCounter) &&
-            string.IsNullOrEmpty(base.Metadata);
-
         public override VersionFormat Format => VersionFormat.Docker;
 
         public override bool IsPrerelease => !string.IsNullOrEmpty(Release) && OriginalString != Latest;
```

Comment:
Does anything need to change here, with `latest` still getting special treatment in terms of pre release determination?

Reply:
You could use latest in a prerelease capacity, and it could be useful to do so. But I would be happy to say that the latest tag in most repositories represents the latest release rather than a work in progress.